### ISSUE TITLE
CSI API group not found

### ISSUE THREAD

**Issue by ArchiFleKs** [vmware-tanzu/velero#5002]
**What steps did you take and what happened:**
[A clear and concise description of what the bug is, and what commands you ran.)

I upgrade to external snapshotter v6.0.1 recently and velero stopped working because of:

```
time="2022-06-14T11:30:53Z" level=fatal msg="The 'EnableCSI' feature flag was specified, but CSI API group [snapshot.storage.k8s.io/v1beta1] was not found." logSource="pkg/cmd/server/server.go:551"
```

**What did you expect to happen:**

Working with v1

**Environment:**

- Velero version (use `velero version`): 1.8.1
- Velero features (use `velero client config get features`): CSI
- Kubernetes version (use `kubectl version`): v1.22.4
- Kubernetes installer & version: EKS
- Cloud provider or hardware configuration: EKS
- OS (e.g. from `/etc/os-release`): Bottlerocket

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by blackpiglet** [comment]
@ArchiFleKs 
This is because Velero v1.8.1 still use snapshot.storage.kubernetes.io/v1beta1, and external-snapshotter v6.0.1 deprecated snapshot.storage.kubernetes/v1beta1:
https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v6.0.1

Velero team already upgrade the snapshot client to snapshot.storage.kubernetes.io/v1, and it will be included in coming v1.9 Velero. It will be released at the end of June. You can also have a try by the main release of Velero as a temporary solution.

**Comment by ArchiFleKs** [comment]
Thanks, yes for now I rollback the csi snapshotter. Closing this
